src/crimson/os/seastore/seastore.cc

```diff
@@ -2282,16 +2296,22 @@ SeaStore::Shard::tm_ret
 SeaStore::Shard::_omap_set_values(
   internal_context_t &ctx,
   OnodeRef &onode,
-  std::map<std::string, ceph::bufferlist> &&aset)
+  std::map<std::string, ceph::bufferlist> &&aset,
+  const omap_root_le_t &omap_root)
 {
   return _omap_set_kvs(
```

Comment:
Looks `_omap_set_values()` can be easily reused where `_omap_set_kvs()` are called (2 another places).